src/components/data-management/SamplesTable.jsx

```diff
@@ -40,24 +37,22 @@ import { metadataNameToKey, metadataKeyToName, temporaryMetadataKey } from 'util
 import integrationTestConstants from 'utils/integrationTestConstants';
 
 import 'utils/css/data-management.css';
-import fetchAPI from 'utils/http/fetchAPI';
+
 import { ClipLoader } from 'react-spinners';
 import useConditionalEffect from 'utils/customHooks/useConditionalEffect';
 
-const { Paragraph, Text } = Typography;
```

Comment:
Can you see if you can remove `Text` from being used? If we're not using `Text` for anything component specific, I think we can replace it with `span` and remove this import.